server/ai_mediaserver.go

```diff
@@ -593,6 +593,7 @@ func (ls *LivepeerServer) StartLiveVideo() http.Handler {
 				requestID:              requestID,
 				streamID:               streamID,
 				pipelineID:             pipelineID,
+				pipeline:               pipeline,
```

Comment:
Probably need to add the same to [CreateWhip](https://github.com/livepeer/go-livepeer/blob/e6aac56deb788c5528bf335332e6d0227041233b/server/ai_mediaserver.go#L879-L892) as well

Reply:
Ahh, right, I'll add it.